tutorials/cookbooks/running_production.md

```diff
@@ -0,0 +1,308 @@
+
+You can always check the latest MeiliSearch stable version, and get MeiliSearch for the Operating System of your choice, by visiting the following link:  
+
+**[Latest MeiliSearch Stable Version](https://github.com/meilisearch/MeiliSearch/releases/latest)**  
+
+MeiliSearch is finally installed and ready to use. If you want to make it accessible from everywhere in your system, you should move this binary into your system binaries folder, with the following command:  
+
+```bash
+# Move the MeiliSearch binary to your system binaries
+$ mv ./meilisearch /usr/bin/
+```
+
+Now you can test your new installation by running `meilisearch` as a command in your terminal (use ctrl-c or command-c to stop execution):
+
+```bash
+$ meilisearch
+
+# --- Expected output for v0.10.1 ---
+888b     d888          d8b 888 d8b  .d8888b.                                    888
+8888b   d8888          Y8P 888 Y8P d88P  Y88b                                   888
+88888b.d88888              888     Y88b.                                        888
+888Y88888P888  .d88b.  888 888 888  "Y888b.    .d88b.   8888b.  888d888 .d8888b 88888b.
+888 Y888P 888 d8P  Y8b 888 888 888     "Y88b. d8P  Y8b     "88b 888P"  d88P"    888 "88b
+888  Y8P  888 88888888 888 888 888       "888 88888888 .d888888 888    888      888  888
+888   "   888 Y8b.     888 888 888 Y88b  d88P Y8b.     888  888 888    Y88b.    888  888
+888       888  "Y8888  888 888 888  "Y8888P"   "Y8888  "Y888888 888     "Y8888P 888  888
+
+[2020-05-04T11:47:13Z INFO  meilisearch] Database path: "./data.ms"
+[2020-05-04T11:47:13Z INFO  meilisearch] Start server on: "127.0.0.1:7700"
+```
+
+## Step 2: Run MeiliSearch as a service
+
+In Linux environments, a `service` is a process that can be launched when the operating system is booting, and it will keep running in the background. One of its biggest advantages is that it can make your program available at any moment, in a stable and consistent way. Even if it finds some execution problem or crashes, the service will be restarted and your program will be run again.  
+
+> This is not required, but if you are new to services and systemd, you can learn the [basics of Linux services](https://www.hostinger.com/tutorials/manage-and-list-services-in-linux/) here.  
+
+On Debian (and Linux distributions in general), `systemd` allows you to create and manage your own custom services. We want to make sure that MeiliSearch is always responding to your requests. And for that, we are going to build our own service. This way, we will make sure that your MeiliSearch service will always be available and running when the server is on. If any crash occurs during the running of MeiliSearch, or the system reboots for any reason, systemd automatically restarts MeiliSearch for you.  
```

Comment:
Sometimes, "we/us" is used. Sometimes, "you/your". It is better to make a perspective choice and stick to it.
```suggestion
On Debian and other Linux distributions, `systemd` allows you to create and manage your own custom services. In order to make sure that MeiliSearch will always respond to our requests, we are going to build our own service. This way, we will ensure our MeiliSearch service will always be available and running when the server is on. If any crash occurs during the running of our instance, or in case of system reboot, `systemd` will automatically restart it for us.  
```